config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -260,3 +260,51 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  kubernetes/perf-tests:
```

Comment:
you might want to create another subdir as jobs/kubernetes/perf-tests, but, your choice.

Reply:
I think we should generally clean up this dir a bit, but I wouldn't mix that with this PR - we probably need that now as we broke tests too many times already with changes in this repo.